target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh

```diff
@@ -131,7 +131,8 @@ platform_do_upgrade() {
 		CI_KERNPART="linux"
 		nand_do_upgrade "$1"
 		;;
-	cudy,wr3000h-v1)
+	cudy,wr3000h-v1|\
+	acer,predator-w6x)
 		CI_UBIPART="ubi"
 		nand_do_upgrade "$1"
```

Comment:
The default partition of `nand_do_upgrade` is `ubi` and this can be handled by the default section (`*)`) without device-specific entry.

https://github.com/openwrt/openwrt/blob/f9a8a3636c0079d01bcd2ea2458929c2f33a6458/package/base-files/files/lib/upgrade/nand.sh#L12

https://github.com/openwrt/openwrt/blob/f9a8a3636c0079d01bcd2ea2458929c2f33a6458/target/linux/mediatek/filogic/base-files/lib/upgrade/platform.sh#L212-L214